### DISCUSSION TITLE
Please add a feature to rotate / add / remove page (all or individual)

### DISCUSSION THREAD

**Discussion by devitalii** [sumatrapdfreader/sumatrapdf#3311] [Ideas]
Please add a feature to rotate / add / remove page (all or individual).

**Comment by GitHubRulesOK** [comment]
That's possible with Acrobat-Pro Subscription or MuPDF Editing but **SumatraPDF is A viewer** like next weeks *Edge Acrobat Viewer* where you pay for the Edgiting team.

**Reply by TCB13** [reply]
So and why can't we have this in Sumatra? There are other free tools that can do it like PDFfil and PDF Tools.

**Reply by GitHubRulesOK** [reply]
@TCB13 
permanent rotate requires a full editor ability to destruct pages without affecting other objects, like Acrobat Pro its not in their heavyweight Reader, Command line binaries can be a few MB and do such edits but MuPDF core does not modify the guts of a PDF, for that you use MuTool or PyMuPDF.
